spicetify: add page (#8762)
tldr-pages/tldr: pages/common/spicetify.md

# spicetify

> A CLI utility to customize the Spotify client UI and functionality.
> More information: <https://spicetify.app>.

> - Generate config file on very first run and show help text:

Reviewer: ```suggestion
- Generate a configuration file on very first run and display help:
```

`spicetify`

> - Backup and preprocess Spotify app files:

Reviewer: ```suggestion
- Backup and preprocess Spotify application files:
```

`spicetify backup`

> - Print all config fields and values:

Reviewer: We try to avoid abbreviations
```suggestion
- Print all configuration fields and values:
```

`spicetify config`

> - Change the value of a config field:

Reviewer: ```suggestion
- Change the value of a configuration field:
```
